tests/cnf/ran/gitopsztp/internal/helper/imageregistry.go

```diff
@@ -0,0 +1,77 @@
+package helper
+
+import (
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/imageregistry"
+	"github.com/openshift-kni/eco-goinfra/pkg/storage"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/gitopsztp/internal/tsparams"
+	operatorv1 "github.com/openshift/api/operator/v1"
+)
+
+// RestoreImageRegistry restores the image registry with the provided name back to imageRegistryConfig, copying over the
+// labels, annotations, and spec from imageRegistryConfig, then waiting until the image registry is available again.
+func RestoreImageRegistry(
```

Comment:
Wondering why do we create new file for these two helper while reusing the helper.go for serviceaccounts? Just want to have a clear guideline on when to create new file.

Reply:
@yliu127  Looking back on it this probably doesn't need its own file. But the original logic is that these functions both go together but the helper.go functions can't be separated as nicely and I didn't want a waittoexist.go